cmd/context.go

```diff
@@ -169,17 +169,27 @@ func showContext(client *client.Client, vcsType, orgName, contextName string) er
 	return nil
 }
 
-func readSecretValue() (string, error) {
+// ReadSecretValue reads a secret from a buffer
+func ReadSecretValue() (string, error) {
 	stat, _ := os.Stdin.Stat()
+
+	buffSize, err := os.Stdin.Stat()
```

Comment:
To avoid the redundancy, would it be possible to remove this `Stat()` call and instead use the one on line 174?

```suggestion
	stat, err := os.Stdin.Stat()
	if err != nil {
	    return "", err
	}
	
	reader := bufio.NewReaderSize(os.Stdin, int(stat.Size()))
	// ...
```